packages/@aws-cdk/aws-ecs-patterns/lib/ecs/scheduled-ecs-task.ts

```diff
@@ -113,6 +114,14 @@ export class ScheduledEc2Task extends ScheduledTaskBase {
       throw new Error('You must specify a taskDefinition or image');
     }
 
-    this.addTaskDefinitionToEventTarget(this.taskDefinition);
```

Comment:
If you keep the base class as described above, this construct doesn't need to be changed.